core/src/main/scala/org/bitcoins/core/util/FutureUtil.scala

```diff
@@ -1,8 +1,26 @@
 package org.bitcoins.core.util
 
 import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
 
 object FutureUtil {
 
+  /**
+    * Executes a series of futures sequentially
+    *
+    * @param items The elements we want to transform into futures
+    * @param fun A function that transforms each element into a future
+    * @return The processed elements
+    */
+  def sequentially[T, U](items: Seq[T])(fun: T => Future[U])(
```

Comment:
Is this something that exists in 2.12 and not 2.11? Maybe add a comment about it if that is the case

Reply:
It does not exist in 2.12. There's `Future.sequence` that runs a sequence of futures in parallell, but no method for running them one after another